src/node_util.cc

```diff
@@ -219,6 +222,17 @@ void Initialize(Local<Object> target,
 
   env->SetMethod(target, "safeGetenv", SafeGetenv);
 
+#if defined(NODE_REPORT)
+  nodereport::InitializeNodeReport();
+  env->SetMethod(target, "getNodeReport", nodereport::GetReport);
+  env->SetMethod(target, "setNodeReportEvents", nodereport::SetEvents);
+  env->SetMethod(target, "setNodeReportSignal", nodereport::SetSignal);
+  env->SetMethod(target, "setNodeReportFileName", nodereport::SetFileName);
+  env->SetMethod(target, "setNodeReportDirectory", nodereport::SetDirectory);
+  env->SetMethod(target, "setNodeReportVerbose", nodereport::SetVerbose);
+  env->SetMethod(target, "triggerNodeReport", nodereport::TriggerReport);
+#endif  //  NODE_REPORT
```

Comment:
Why are these being exposed both here and in `node_report_module.cc`? These should not be exposed via the `util` internal binding.

Reply:
removed from here and exposed through `util`. However, subsequent reviews and changes thereon made some of these interfaces change their name and behavior.